packages/govern/internal/actions/DaoAction.ts

```diff
@@ -35,6 +54,6 @@ export default class DaoAction extends AbstractAction {
   public async execute(): Promise<Dao> {
```

Comment:
We can remove this overwrite at all. This was only required because I was formating the output.

Reply:
I remove the format() call instead and just use execute() to format the output as the original design, which is better as it's less code..